Child devel cd 2

callid01 opened the pull request.

Jen-e2 commented:
Hiya.

The tab looks really good! 

Just a couple of points, let me know if you have any questions:

- When you've moved to the ethnicity breakdown and the geog changes to just Scotland, it only changes back when you move to yearly total to including health board and council area. So after ethnicity, if I move to sex or SIMD breakdowns I can only view Scotland levels. 

- The ethnicity labels, just wanted to check if they are finalised as 'Pole' is included for example? I'm not 100% sure so correct me if I'm wrong, that could be my knowledge 

- In the plot description, the line describing the colours of the geography's at yearly total describes Scotland as the baseline for HB and Scotland comparison, but the council area as baseline for council and HB. Hope this makes sense. Just wondered if HB was meant to be baseline for the later? 

Looks super good though, you've included a lot !!

callid01 commented:
Thanks so much for checking!

- I've changed the dropdowns so you can select HB/LA for sex/SIMD breakdowns after being on ethnicity, although this also removes the geography selected when moving between breakdowns :( Do you know of a way around that?
- Thanks for flagging the ethnicities - I've updated them
- I think the plot description did use HB as the baseline but it wasn't very clear, I've reworded so hopefully is now but let me know if not!
- Also it looked like one of the geography look up files was updated - I've amended the master data transfer script to locate the old file in the archive rather than updating to the new file in case it was different, hope that's ok!

Jen-e2 commented:
The changes look fab! 

I added an argument to the select input update bit so that it selects what is currently selected (if that makes sense). It works for me if you want to give it a double check and see if you are happy with it. 

I'll approve the pull request and you can then merge when ready :)
